src/themes/__tests__/ThemeProvider.spec.js

```diff
@@ -22,6 +22,13 @@ import Sample from '../../website/app/demos/ThemeProvider/components/Sample';
 import examples from '../../website/app/demos/ThemeProvider/examples';
 import testDemoExamples from '../../../utils/testDemoExamples';
 
+// Allow full theme values in snapshots for ThemeProvider
```

Comment:
It would be nice to have this logic live inside `themeSerializer`, but I didn't see a good way to do so.  I'm open to suggestions or assistance.

Reply:
Agreed, but I'm not seeing a way, either.